constant_testing

```diff
@@ -110,17 +106,60 @@ clear_screen() {
   tput reset
 }
 
+elm_test_command() {
+  echo "elm-test"
+}
+
+elm_test() {
+  elm_json_path=$(find_first_elm_json)
+  elm_json_dir=$(dirname $elm_json_path)
+  pushd $elm_json_dir > /dev/null
+  run_command $(elm_test_command) "Your elm tests passed! First time for everything..." "Your elm is riddled with bugs... (at least you have tests(s) though)..." true
+  popd > /dev/null
+}
+
+run_command() {
+  cmd="$1"
+  success_message="$2"
+  failure_message="$3"
+  echo_output=$4
+  log_info "Running: $cmd"
+  output=`$cmd`
+  result=$?
+  last_run_cmd=$cmd
+  if [[ $echo_output = true ]]; then
+    echo "$output"
+  fi
+  if [[ $result != 0 ]]; then
+    log_failure "$failure_message"
+  else
+    log_success "$success_message"
+  fi
+}
+
+run_last_cmd() {
```

Comment:
i dont think it should do this if you have a pinned file?

i.e. if you run 

constant_testing chuffy_test.exs
 
it should always run that no matter what file you save?